sdk/core/Azure.Core/src/HttpMessage.cs

```diff
@@ -16,6 +16,7 @@ public sealed class HttpMessage : IDisposable
     {
         private ArrayBackedPropertyBag<ulong, object> _propertyBag;
         private Response? _response;
+        private bool _ownsResponse;
```

Comment:
why do we need this? Can't we simply set the _response field to null when we transfer ownership?

Reply:
You are correct